docs/README.md

```diff
@@ -6,35 +6,35 @@
 
 ## What is refgenie?
 
-Refgenie is full-service reference genome manager. It provides command-line and python interfaces to download pre-built genome assets like indexes. It can also build assets for custom genomes.
+Refgenie is full-service reference genome manager. It provides command-line and python interfaces to download pre-built reference genome "assets" like indexes used by different bioinformatics tools. It can also build assets for custom genome assemblies, and it facilitates systematic organization of, and access to, local genome "assets."
 
 ## What makes refgenie better?
 
 Refgenie provides programmatic access to a standard genome folder structure, so that software can easily swap from one genome to another. There are other similar projects, but Refgenie has a few advantages:
 
-1. **It provides a command-line interface to download individual resources**. Think of it as `GitHub` for reference genomes. You just type `refgenie pull -g hg38 -a bowtie2_index`.
+1. **It provides a command-line interface to download individual resources**. Think of it as `GitHub` for reference genomes. You just type `refgenie pull -g hg38 -a kallisto`.
 
```

Comment:
the asset should be called `kallisto_index` -- `kallisto` would be the tool that consumes the index.

Reply:
similar situation to: https://github.com/databio/refgenie/pull/28#discussion_r293025543